share/eds/byzantine/share_proof_test.go

```diff
@@ -0,0 +1,81 @@
+package byzantine
```

Comment:
Can you pls use `git mv` or similar to make the diff shorter so it's shown as a file move and not as a new file

P.S. I just realized that we should have done this for all previous refactorings because there it overwrites all the history.

Reply:
Pls also check this for other files as well

Reply:
Initially I got super excited about your suggestion, but seems it won't do the magic. According to [this discussion](https://stackoverflow.com/questions/1094269/whats-the-purpose-of-git-mv), the `git mv` command is just a shorthand for deleting and re-adding. Still, history is accessible  by `git blame`  as long as the add+delete happens in the same commit.

It would be nice to make git track changes as moved files instead of new lines.